sdk/communication/azure-communication-sms/azure/communication/sms/_sms_client.py

```diff
@@ -72,31 +78,55 @@ def from_connection_string(cls, conn_str,  # type: str
         return cls(endpoint, access_key, **kwargs)
 
     @distributed_trace
-    def send(self, from_phone_number, # type: ~azure.communication.sms.PhoneNumberIdentifier
-             to_phone_numbers, # type: list[~azure.communication.sms.PhoneNumberIdentifier]
+    def send(self, from_, # type: str
+             to, # type: Union[str, List[str]]
              message, # type: str
-             **kwargs  #type: Any
-        ): # type: (...) -> SendSmsResponse
+             **kwargs #type: Any
+        ): # type: (...) -> [SmsSendResult]
         """Sends SMSs to phone numbers.
 
-        :param from_phone_number: the sender of the SMS.
-        :type from_phone_number: ~azure.communication.sms.PhoneNumberIdentifier
-        :param to_phone_numbers: the list of recipients of the SMS.
-        :type to_phone_numbers: list[~azure.communication.sms.PhoneNumberIdentifier]
+        :param str from_: The sender of the SMS.
+        :param to: The single recipient or the list of recipients of the SMS.
+        :type to: Union[str, List[str]]
         :param str message: The message in the SMS
-        :keyword send_sms_options: the options object to configure delivery reporting.
-        :type send_sms_options: ~azure.communication.sms.models.SendSmsOptions
-        :return: The response object with the message_id
-        :rtype: SendMessageResponse: ~azure.communication.sms.models.SendMessageResponse
+        :keyword bool enable_delivery_report: Enable this flag to receive a delivery report for this
+         message on the Azure Resource EventGrid.
+        :keyword str tag: Use this field to provide metadata that will then be sent back in the corresponding
+         Delivery Report.
+        :return: A list of SmsSendResults.
+        :rtype: [~azure.communication.sms.models.SmsSendResult]
         """
 
-        send_sms_options = kwargs.pop('send_sms_options', None)
+        enable_delivery_report = kwargs.pop('enable_delivery_report', False)
+        tag = kwargs.pop('tag', None)
+
+        sms_send_options = SmsSendOptions(
+            enable_delivery_report=enable_delivery_report,
+            tag=tag
+        )
 
         request = SendMessageRequest(
-            from_property=from_phone_number.phone_number,
-            to=[p.phone_number for p in to_phone_numbers],
+            from_property=from_,
+            sms_recipients=[
+                SmsRecipient(
+                    to=p,
+                    repeatability_request_id=str(uuid4()),
+                    repeatability_first_sent=datetime.now()
```

Comment:
Anything needed to make it cultural neutral or is that not a problem?

Reply:
Yes, this should be ```datetime.utcnow()``` , so it uses a standard time zone. We're also using UTC in other languages.